### DISCUSSION TITLE
[FEAT] Community Feature - Axolotls

### DISCUSSION THREAD

**Discussion by Netherzapdos** [sunflower-land/sunflower-land#1623] [Ideas]

**Comment by esp123** [comment]
Very detailed proposal, LFG!! Can't wait for the release and its future! 🔥🔥🔥🇵🇭🇵🇭🇵🇭

**Comment by digital-farms** [comment]
Yoooo, How long I've been waiting for this😍

**Comment by PeachNooter** [comment]
Amazing and detailed proposal! Really hope this gets green lit <3

**Comment by TheDcol** [comment]
Yes yes yes

**Comment by ShiningMinions** [comment]
This is amazing and it's clearly thought-out and well planned. I can already picture SFL being a game where you go out in the ocean,  explore and find a ton of awesome mini-projects like these.

**Comment by IvanSkazkin** [comment]
**Yes**

**Comment by SteveChickenCX** [comment]
This is the feature which I'm the most excited for! ♥

**Comment by adamhannigan** [comment]
Looking forward to seeing progress on this 💪 

I would like to call out the idea of having a standalone island that players can visit for this feature alone. Well done 👏 This feels like a much more scalable approach when we have a range of community projects
